src/extensions/registries/entity-setting-registry.ts

```diff
@@ -47,7 +43,7 @@ export class EntitySettingRegistry extends BaseRegistry<EntitySettingRegistratio
       });
     }
 
-    return items.sort((a, b) => b.priority - a.priority);
+    return items.sort((a, b) => (b.priority ?? 50) - (a.priority ?? 50));
```

Comment:
I guess there's no initializer where the default value can be set, instead of doing this check during sorting? Gone are the days of optimizing code :)

Reply:
Oh we could do that instead. Might be better. I would consider that a chore. Will submit a PR (but I think this should be in the next alpha).